backend/controllers/user.controller.js

```diff
@@ -1,79 +1,10 @@
 const passport = require('passport');
 const authUtils = require('../utils/auth.js');
-const User = require('../models/user.model')
```

Comment:
please restore the functions you deleted from this file, this is breaking the application.